### PR TITLE
Add install/uninstall/contribution guides

### PR DESCRIPTION
**Pull request by dhaavi**

### CODE REVIEW

File: portmaster/install/mobile.md [hunk 1]
```diff
@@ -3,25 +3,27 @@ title: Status of Mobile
...
-While this will comes with a multitude of challanges, our dream is to support as many mobile platforms as possible.  
-
 This page gives an overview over the current status of supporting mobile platforms.
-If you are interested in what we are currently working on and what is planned for the near future, check out our [next page](https://safing.io/next/).
+
+**⚠️ The Portmaster is currently not available for any mobile platforms.**
+
+While we know supporting mobile will come with a multitude of challenges, it still remains our dream to support many mobile platforms in the future. Currently, these tasks [remain in our backlog](https://safing.io/backlog/).
+
+[You can always see what we are currently working on and what is planned for the near future on the Next Page](https://safing.io/next/).
```
Comment (dhaavi) [inline portmaster/install/mobile.md:17]: I dislike such long links as they decrease readability and do not communicate what exactly one will find when clicking it. I'd recommend only putting the link on `Next Page`.

File: portmaster/install/mobile.md [hunk 2]
```diff
@@ -3,25 +3,27 @@ title: Status of Mobile
...
 
 __Distribution__
 
-The Portmaster puts you back in control.
-We think that this might be somewhat against the interests of Google, and the Play store has banned ad blockers in the past.
+The Portmaster will put you back in control of your mobile phone and all of its network connections.
+We think that this might be against the interests of Google, and the Play store has banned ad blockers in the past.
 
-This means that the Portmaster may only be available via alternative app stores, such as F-Droid.
+This means that the Portmaster will likely only be available via alternative app stores, such as F-Droid or via direct `.apk` distributions
```
Comment (dhaavi) [inline portmaster/install/mobile.md:31]: Missing final period.

File: portmaster/install/windows.md [hunk 3]
```diff
@@ -33,3 +33,10 @@ This is done by setting the registry key `HKEY_LOCAL_MACHINE\SYSTEM\CurrentContr
 You can easily uninstall the Portmaster by using the system preferences or by executing the `portmaster-uninstaller.exe`, which you can find in `C:\ProgramData\Safing\Portmaster`.
 
 The uninstaller reverts all steps mentioned above and also requires a reboot afterwards.
+
+
+#### Known Issues
+
+**Notifications are not working on Windows 7 and 8.**
+
+Microsoft added a notification service that supports actions with Windows 10 so Portmaster's notifications won't work on Windows 7 or 8. We will push alternatives as we find solutions and time to do so. In the meantime we recommend not to use the "ask" action as a default when running on Windows 7 or 8.
```
Comment (dhaavi) [inline portmaster/install/windows.md:42]: We now call it the "prompt" action.

**Review by dhaavi — COMMENTED** [review]
LGTM so far

File: portmaster/guides/contribute.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Help make the Portmaster better for everyone. [Download](https://safing.io/portmaster/#downloads) and use the Portmaster, [report bugs](https://github.com/safing/portmaster/issues/new?template=bug-report.md) as you stumble upon them. And as you get to know the software, [suggest improvements and new features](https://github.com/safing/portmaster/issues/new?template=feature.md).
+
+### Report Linux Distribution Compatibility
+
+Help us understand Portmaster compatibility with different Linux distributions. Whether it works with your distro or not - your reports will help us massively to understand where we stand. We will update the [Linux compatibility list](https://docs.safing.io/portmaster/install/linux#compatibility) accordingly.
+
+### Report VPN Compatibility
+
+Help us understand Portmaster compatibility with different Linux distributions.
+
```
Comment (dhaavi) [inline portmaster/guides/contribute.md:24]: Some copy pasta leftovers?

**Review by dhaavi — COMMENTED** [review]
LGTM, requesting minor changes.

**Review by dhaavi — COMMENTED** [review]
LGTM!

**Comment by davegson** [comment]
everything except the last commit got reviewed by @dhaavi - the last commit was basically copying the current Wiki VPN Compatibility page and adding it to the docs. Good enough fix for now. Later on, we will add templates for Compatibility reports and add an overview table for in/compatible VPN clients

**Pull request merged**